graphql/course/mutations.ts

```diff
@@ -82,6 +82,19 @@ export class MutateCourseResolver {
         return result;
     }
 
+    @Mutation((returns) => GraphQLModel.Course)
+    @Authorized(Role.ADMIN, Role.INSTRUCTOR)
```

Comment:
This should be `@AuthorizedDeferred` as we do not evaluate the roles before the method, but inside of it (with `hasAccess`). Roles should be `Role.ADMIN` and `Role.OWNER` - We want only the owner of a course be able to edit their own course, not every course instructor should be able to edit every course.